packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -972,7 +972,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         const requestParser = RequestParser.create(request);
         const id = requestParser.pathParts[0];
 
-        if (id === "_channel") {
```

Comment:
this path wasn't being hit yet, right? (looks like you didn't add any cases in the previous change #3888)

Reply:
Correct, I paused remaining work, now trying to get back :)